app/frontend/src/components/forms/manage/ManageVersions.vue

```diff
@@ -49,18 +49,23 @@
       <template #[`item.create`]="{ item }">
         <v-tooltip bottom>
           <template #activator="{ on, attrs }">
-            <v-btn
-              color="primary"
-              text
-              small
-              @click="createVersion(item.formId, item.id)"
-              v-bind="attrs"
-              v-on="on"
-            >
-              <v-icon>add_circle</v-icon>
-            </v-btn>
+            <div v-bind="attrs" v-on="on">
```

Comment:
clicking this 'add' button isnt working for me. maybe these attributes need to be on the v-btn?

Reply:
yikes, local change I'd forgot to push that fixes this. Thanks!